tests/lib/rules/require-key-for-conditionally-rendered-repeated-components.js

```diff
@@ -0,0 +1,313 @@
+/**
+ * @author Felipe Melendez
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+const RuleTester = require('eslint').RuleTester
+const rule = require('../../../lib/rules/v-if-else-key')
+
+const tester = new RuleTester({
+  parser: require.resolve('vue-eslint-parser'),
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: 'module'
+  }
+})
+
+tester.run('v-if-else-key', rule, {
+  valid: [
```

Comment:
Please add the following valid test case:

```html
<div v-if="foo" />
<span v-else />
```

Reply:
[d1b5039](https://github.com/vuejs/eslint-plugin-vue/pull/2280/commits/d1b5039d8213b6a3e91159b53ad0f84b85cb6f13)